File: components_page/components/layout/order_offset.R
library(dashBootstrapComponents)
library(dashHtmlComponents)

row <- htmlDiv(
  list(
    dbcRow(
      dbcCol(
        htmlDiv("A single, half-width column"),
        width = list(size = 6, offset = 3)
      )
    ),
    dbcRow(
      list(
        dbcCol(
          htmlDiv("The last of three columns"),
          width = list(size = 3, order = "last", offset = 1)
        ),
        dbcCol(
          htmlDiv("The first of three columns"),
          width = list(size = 3, order = 1, offset = 2)
        ),
        dbcCol(
          htmlDiv("The second of three columns"),
          width = list(size = 3, order = 5)
        )
      )
    )
  )
)

File: docs/components_page/components/layout/order_offset.py
import dash_bootstrap_components as dbc
from dash import html

row = html.Div(
    [
        dbc.Row(
            dbc.Col(
                html.Div("A single, half-width column"),
                width={"size": 6, "offset": 3},
            )
        ),
        dbc.Row(
            [
                dbc.Col(
                    html.Div("The last of three columns"),
                    width={"size": 3, "order": "last", "offset": 1},
                ),
                dbc.Col(
                    html.Div("The first of three columns"),
                    width={"size": 3, "order": 1, "offset": 2},
                ),
                dbc.Col(
                    html.Div("The second of three columns"),
                    width={"size": 3, "order": 5},
                ),
            ]
        ),
    ]
)
